packages/flow_parser/flow_parser.0.36.0/opam

```diff
@@ -0,0 +1,21 @@
+opam-version: "1.2"
+maintainer: "gabe@fb.com"
+authors: [
+  "Avik Chaudhuri"
+  "Basil Hosmer"
+  "Gabe Levi"
+  "Jeff Morrison"
+  "Marshall Roch"
+  "Sam Goldman"
+  "James Kyle"
+]
+homepage: "https://github.com/facebook/flow/tree/master/src/parser"
+bug-reports: "https://github.com/facebook/flow/issues"
+license: "BSD3"
+
+build: [ "sh" "-c" "cd src/parser && make ocamlfind-install" ]
```

Comment:
something that helps with error recovery in opam is to have a separate `build` and `install` step; splitting this in future revs would be nice if the upstream build system supports it